source/plugins/go/src/extension/extension.go

```diff
@@ -94,6 +111,17 @@ func getDataTypeToStreamIdMapping() (map[string]string, error) {
 			logger.Printf("Info::mdsd::datatype: %s, outputstreamId: %s", dataType, outputStreamID)
 			datatypeOutputStreamMap[dataType] = outputStreamID.(string)
 		}
+		
+		//fetch dataCollectionSettings from extensionSettings and set to datatypeOutputStreamMap
+		extensionSettings := extensionConfig.ExtensionSettings
+		dataCollectionSettings := extensionSettings["dataCollectionSettings"]
+		containerLogV2Flag, isContainerLogV2FlagExists := dataCollectionSettings["containerLogV2"]

```

Comment:
What is the value of isContainerLogV2FlagExists?

Reply:
This is  a variable declaration with an assignment, 'isContainerLogV2FlagExists' will be true if dataCollectionSettings contains the key "containerLogV2"